packages/decentraland-ecs/src/cli/mock-catalyst/index.ts

```diff
@@ -0,0 +1,124 @@
+import * as path from 'path'
+import { createProxyMiddleware } from 'http-proxy-middleware'
+import * as express from 'express'
+import { getSceneJson } from '../setupUtils'
+import { getAllPreviewWearables } from '../wearables'
+
+export const mockCatalyst = (app: express.Application, baseFolders: string[], rootFolder: string) => {
+  serveFolders(app, baseFolders, rootFolder)
+  app.get('/lambdas/explore/realms', (req, res) => {
+    res.json([
+      {
+        serverName: 'localhost',
+        url: `http://${req.get('host')}`,
+        layer: 'stub',
+        usersCount: 0,
+        maxUsers: 100,
+        userParcels: []
+      }
+    ])
+  })
+
+  app.get('/lambdas/contracts/servers', (req, res) => {
+    res.json([
+      {
+        address: `http://${req.get('host')}`,
+        owner: '0x0000000000000000000000000000000000000000',
+        id: '0x0000000000000000000000000000000000000000000000000000000000000000'
+      }
+    ])
+  })
+
+  app.get('/lambdas/profiles', async (req, res, next) => {
+    try {
+      const previewWearables = await getAllPreviewWearables({
+        baseFolders,
+        catalystRootFolder: rootFolder,
+        baseUrl: ''
+      }).map(($) => $.id)
```

Comment:
can we change the generic name `$` to something more related to the prop that we are iterating ?
$ sounds like jquery 😛 
.map(wearable => wearable.id)